source/FreeRTOS_ND.c

```diff
@@ -1059,6 +1059,18 @@
                    {
                        size_t uxICMPSize;
                        BaseType_t xCompare;
+                       const NetworkEndPoint_t * pxTargetedEndPoint = pxEndPoint;
+                       const NetworkEndPoint_t * pxEndPointInSameSubnet = FreeRTOS_InterfaceEPInSameSubnet_IPv6( pxNetworkBuffer->pxInterface, &( pxICMPHeader_IPv6->xIPv6Address ) );
```

Comment:
What will happen if we have more than one endpoint in same subnet?
Is it necessary to travese all of them?